packages/fxa-auth-server/lib/routes/session.js

```diff
@@ -344,7 +342,7 @@ module.exports = function (
           details: {
             accountEmailVerified,
             sessionVerificationMethod,
-            sessionVerificationSuccessful,
+            sessionVerified,
```

Comment:
Personally, I preferred the previous naming... I think it's more clear, more consistent, and it less likely to be confused with the 'sessionVerfied' flag that is scattered all over.